pkg/models/terminal/terminal.go

```diff
@@ -53,6 +57,10 @@ type TerminalSession struct {
 	sizeChan chan remotecommand.TerminalSize
 }
 
+var NodeSessionCounter = make(map[string]*int64)
```

Comment:
thanks, i'll modify it and use sync.Once to init

Reply:
Also add a cap to this map, since the nodes can be listed.  We can get the max nodes num.

Reply:
> Also add a cap to this map, since the nodes can be listed. We can get the max nodes num.

Maybe we need to list all nodes and pre-set the nodename to this map ?